src/app/screens/App/screens/Home/components/FilterSelect.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./FilterSelect.css";
+
+export default function FilterSelect(props) {
+  return (
+    <select
+      name="filterField"
+      onChange={props.onChange}
+      className="filter-select"
+      defaultValue=""
+    >
+      <option disabled hidden value="">
+        Seleccionar filtro
```

Comment:
In wolox we use strings from a string files for 3 reasons
1- Strings are centralized (global or local)
2- Is easy add translations later.
3- render components are calling many times and every render call create a new STRING instance, so if we use a string from a string file the render not create a new instance, use just a reference and this save a little of memory. 

Structure folder:
```
screens
   - home
      - index.js
      - layout.js
      - strings.js
      - styles.css
```

Reply:
follow this for all strings that you have to show on the view.